lib/config.js

```diff
@@ -298,17 +84,6 @@ config.views.browserVersions = {
 };
 
 config.views.vars = {
```

Comment:
Can we get rid of all these view vars as well? And remove the "data" thing we pass to the frontend?

Reply:
+1 to that!  Which I believe would eliminate that weird getViewsVars thing that links bedrock-rest to this module requiring bedrock-view and all it's deps to be installed wherever bedrock-rest is used.

Reply:
Removed.  Will make PRs for other packages that I know use this.